pkg/cmd/install/hubaddon/scenario/addon/policy/addon-controller_configmap.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright Contributors to the Open Cluster Management project
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: governance-policy-addon-controller
+  name: policy-addon-ctrl-manager-config
+  namespace: open-cluster-management
+data:
+  controller_manager_config.yaml: |
```

Comment:
I don't think we use this configmap - I think we would need to make changes in the addon-controller to accept configuration like this, as opposed to (or in addition to) the command line flags. See https://book.kubebuilder.io/component-config-tutorial/tutorial.html

Reply:
Deleting...